xsc/services/analyticsevent.go

```diff
@@ -0,0 +1,140 @@
+package services
```

Comment:
This file name should be renamed to `analytics.go`, rather than `analyticsevent.go`, to be consistent with its nabouring `version.go` file.